diff --git a/openscad.pro b/openscad.pro
index b76ca76..13f0a2d 100755
--- a/openscad.pro
+++ b/openscad.pro
@@ -577,8 +577,7 @@ SOURCES += src/ext/lodepng/lodepng.cpp
 HEADERS += src/ext/lodepng/lodepng.h
            
 # ClipperLib
-SOURCES += src/ext/polyclipping/clipper.cpp
-HEADERS += src/ext/polyclipping/clipper.hpp
+LIBS += -lpolyclipping
 
 # libtess2
 INCLUDEPATH += src/ext/libtess2/Include
diff --git a/src/clipper-utils.h b/src/clipper-utils.h
index d961830..d2ed22f 100644
--- a/src/clipper-utils.h
+++ b/src/clipper-utils.h
@@ -1,6 +1,6 @@
 #pragma once
 
-#include "ext/polyclipping/clipper.hpp"
+#include <polyclipping/clipper.hpp>
 #include "Polygon2d.h"
 
 namespace ClipperUtils {
diff --git a/src/libsvg/shape.h b/src/libsvg/shape.h
index f139f28..0b00899 100644
--- a/src/libsvg/shape.h
+++ b/src/libsvg/shape.h
@@ -34,7 +34,7 @@
 #include <Eigen/Geometry>
 
 #include "util.h"
-#include "ext/polyclipping/clipper.hpp"
+#include <polyclipping/clipper.hpp>
 
 namespace libsvg {
 